new-dti-website/components/navbar.tsx

```diff
@@ -70,14 +70,15 @@ const Navbar: React.FC = () => {
         </div>
         <div className={`flex lg:hidden w-fit ${isMenuOpen ? 'z-50' : 'z-10'}`}>
           {!isMenuOpen ? (
-            <Image
-              className="h-12 w-auto md:h-14"
-              src="/icons/hamburger_icon.svg"
-              width={56.5}
-              height={56.5}
-              alt="Hamburger Menu Icon"
-              onClick={(e) => handleMenuClick()}
-            />
+            <button onClick={handleMenuClick}>
+              <Image
+                className="h-12 w-auto md:h-14"
+                src="/icons/hamburger_icon.svg"
+                width={56.5}
+                height={56.5}
+                alt="Hamburger Menu Icon"
+              />
+            </button>
```

Comment:
For icon buttons like this one, we should be adding `aria-label` attribute for accessibility. It helps screen reader technologies announce to the user what this icon button does (since it doesn't have a text label).

[More on `aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)

When you add `aria-label`, I think we won't need the alt text on `<Image>` anymore since that would be redundant. However, make sure to still include the empty `alt=""` attribute to explicitly tell screen readers that this image is decorative and can be safely skipped. (If we forget to add the attribute, even if we don't technically need it, the accessibility score for the website is lowered and can be flagged on automated testing tools)